app/src/main/res/layout/music_queuepage.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="15dp">
+
+    <ImageView
+        android:id="@+id/queueCoverArt"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:minWidth="50dp"
+        android:minHeight="50dp"
+        android:src="@drawable/ic_notify"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/queueTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="18sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/queueCoverArt" />
+
+    <TextView
+        android:id="@+id/queueSubtitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="14sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/queueTitle" />
+
+    <android.support.v4.widget.SwipeRefreshLayout
+        android:id="@+id/listQueueRefresh"
+        android:layout_width="wrap_content"
```

Comment:
Set the width for this and the contained RecyclerView and the music_queue_listitem to be match_parent, which means the entire width will be clickable instead of just the text

Reply:
Fixed the width of these to be match_parent. Layout formatting was giving me a hard time with these so kind of got it working and didn't want to touch it in fear it would break again :)

Reply:
I'm the same way! I'm much more comfortable with the APIs than the UIs :)